doc/webui/README.md

```diff
@@ -132,56 +125,37 @@ API to allow instances to register themselves with `clusterd`
 ##### list all instances [GET]
 
 + Response 200 (application/json)
-
-        [
-            {
-                "id": "507f191e810c19729de860ea",
-                "name": "test instance",
-                "host": "192.168.0.5"
-            },
-            {
-                "id": "507f191e810c19729de860eb",
-                "name": "test instance #2",
-                "host": "192.168.0.6"
-            }
-        ]
+    + Attributes (array[Instance])
 
 ##### create a new instance [POST]
 
++ Attributes (Base Instance)
+
```

Comment:
That's how it was done in the [example](https://raw.githubusercontent.com/apiaryio/api-blueprint/master/examples/10.%20Data%20Structures.md)